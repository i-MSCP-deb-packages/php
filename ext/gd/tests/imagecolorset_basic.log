
---- EXPECTED OUTPUT
6f2002aafb57b2d275fad6a6258d7476
---- ACTUAL OUTPUT
61915dd4be312a7c7f8a089e0d96253e
---- FAILED
