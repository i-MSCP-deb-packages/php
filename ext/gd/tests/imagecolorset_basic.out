61915dd4be312a7c7f8a089e0d96253e